app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -708,6 +728,42 @@ class StatePlayerRecyclerViewAssembler private constructor(
     }
   }
 
+  private fun createBannerConfetti() {
+    val x = bannerConfettiView!!.width / 3
+    val y = bannerConfettiView!!.height / 2
+    val colorsList = confettiColors.map { getColor(bannerConfettiView.context, it) }
+    val speedMin = 2f
+    val speedMax = 4f
+    val timeToLive = 2000
+    val shapesArray = arrayOf(Square, Circle)
+    val size = Size(7)
+    val sizeWithMass = Size(7, 3f)
```

Comment:
What's the purpose of this size vs. the above? Also, suggest named arguments here since it's not clear what the 3 vs. 7 is.

Reply:
This adds mass to the confetti particle and changes the way the confetti pieces can move in the animation.

Reply:
Added a comment to clarify this